.changeset/three-avocados-search.md

```diff
@@ -0,0 +1,5 @@
+---
+"@rocket.chat/meteor": major
+---
+
+Fixed archived rooms returned in `rooms.autocomplete.channelAndPrivate` endpoint
```

Comment:
the description does not explain to me what changes for this to be a break change